From: Petr Mladek <pmladek@suse.com>
Date: Fri, 10 Jul 2020 11:44:32 +0200
Subject: [PATCH 05/25] MAINTAIERS: Add John Ogness as printk reviewer
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/5.9/older/patches-5.9.1-rt20.tar.xz

John Ogness has started major rework of the printk code. Add him
as reviewer so that he is aware of all other coming changes and
could influence their integration.

Signed-off-by: Petr Mladek <pmladek@suse.com>
Acked-by: Steven Rostedt (VMware) <rostedt@goodmis.org>
Acked-by: Sergey Senozhatsky <sergey.senozhatsky@gmail.com>
Acked-by: John Ogness <john.ogness@linutronix.de>
Link: https://lore.kernel.org/r/20200710094432.19655-1-pmladek@suse.com
Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
---
 MAINTAINERS |    1 +
 1 file changed, 1 insertion(+)

--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -13960,6 +13960,7 @@ PRINTK
 M:	Petr Mladek <pmladek@suse.com>
 M:	Sergey Senozhatsky <sergey.senozhatsky@gmail.com>
 R:	Steven Rostedt <rostedt@goodmis.org>
+R:	John Ogness <john.ogness@linutronix.de>
 S:	Maintained
 F:	include/linux/printk.h
 F:	kernel/printk/
